cob_bringup/robots/raw3-3.xml

```diff
@@ -3,6 +3,7 @@
 
 	<!-- args -->
 	<arg name="robot" value="raw3-3"/>
+	<arg name="include_non_released_packages" default="false"/>
```

Comment:
This has been done according to @ipa-fmw's comment.
So, what should we do? The current would be a more general approach, the other a more clear one.
I don't mind ;-)